che-core-git-impl-jgit/src/main/resources/org.eclipse.che.git.impl.jgit/messages.properties

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Properties file pthname is invalid, it should be: `che-core-git-impl-jgit/src/main/resources/org/eclipse/che/git/impl/jgit/messages.properties`